docs/source/index.rst

```diff
@@ -18,3 +21,19 @@ Indices and tables
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+
+Building the documentation
```

Comment:
Instead of duplicating this, I think linking to the repo README section is enough.